docs/tutorials/notebooks/tracking_progress.ipynb

```diff
@@ -7,21 +7,15 @@
     "tags": []
```

Comment:
<p>Would remove <code>initializer</code>(I know you mean <code>__init__</code> ), since initializers for Sinkhorn can also mean the initial value for potentials/scalings.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/351/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>